bin/viirs2pps.py

```diff
@@ -41,6 +41,9 @@
                         help="Output directory where to store the level1c file")
     parser.add_argument('--iband', action='store_true',
                         help="Iband resolution include I01-I03, M15-M16 and optional M09, M14")
+    parser.add_argument('--reader', type=str,  nargs='?',
```

Comment:
Do we really handle nargs (multiple readers) and if, how does it work? Try the first one first?

Reply:
You are correct. I have misundertsood it.